WarehouseManagement.Core/Services/DeliveryService.cs

```diff
@@ -149,6 +149,8 @@ public async Task<PageDto<DeliveryDto>> GetAllAsync(PaginationParameters paginat
                         MarkerName = dm.Marker.Name
                     })
                     .ToList(),
+                EntriesFinishedProcessingDetails = GetEntriesProcessingDetails(d.Entries, true),
+                EntriesWaitingProcessingDetails = GetEntriesProcessingDetails(d.Entries, false),
```

Comment:
Here both entries which are waiting and are in processing will be seeded, because of `e.FinishedProcessing == null` in `GetEntriesProcessingDetails`. Do we want that or should we add another property `EntriesInProcessingDetails`?

Reply:
No, we want waiting and processing to be in the same object.